Cargo.toml

```diff
@@ -81,6 +81,7 @@ ignore = "0.4"
 include_dir = "0.7"
 indicatif = "0.17"
 indoc = "2"
+inquire = "0.7.5"
```

Comment:
May I ask why is this using `inquire` instead of `dialoguer`?

`inquire` is based off `crossterm`, not `console`, so we have two crates for console APIs that are incompatible, and `scarb-ui` is doing some initial init for `console`. And, we're basically duplicating code here.

Reply:
Will switch to `dialoguer`